libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -3762,6 +3769,81 @@ private static IEnumerable<object> Flatten(IEnumerable<object> list, int dept)
                     },
                     ReturnType.Number,
                     expr => ValidateArityAndAnyType(expr, 1, 1, ReturnType.String)),
+                new ExpressionEvaluator(
+                    ExpressionType.TicksToDays,
+                    (expr, state, options) =>
+                    {
+                        object value = null;
+                        string error = null;
+                        IReadOnlyList<object> args;
+                        long ticksPerDay = 864000000000;
+                        (args, error) = EvaluateChildren(expr, state, options);
+                        if (error == null)
+                        {
+                            if (args[0].IsInteger())
+                            {
+                                value = Convert.ToDouble(args[0]) / ticksPerDay;
+                            }
+                            else
+                            {
+                                error = $"{expr} should contain an integer of ticks";
+                            }
+                        }
+
+                        return (value, error);
+                    },
+                    ReturnType.Number,
+                    expr => ValidateUnaryNumber(expr)),
```

Comment:
just `ValidateUnaryNumber` is ok, no need to apply the whole method with `expr => ValidateUnaryNumber(expr)`

Reply:
changed